src/inputs/TextField/index.tsx

```diff
@@ -1,25 +1,35 @@
 import React from 'react';
 import TextFieldMui from '@material-ui/core/TextField';
-//import { withStyles } from '@material-ui/core/styles';
+import InputAdornment from '@material-ui/core/InputAdornment';
 import styled from 'styled-components';
 
-//import theme from '../../theme';
-
 type Props = {
   value: string;
   onChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
   label: string;
   readOnly?: boolean;
   meta?: any;
   input?: any; // added for compatibility with react-final-form
+  startAdornment?: React.ReactElement;
+  endAdornment?: React.ReactElement;
 } & React.InputHTMLAttributes<HTMLInputElement>;
 
-const CustomTextField = styled(({ input, ...props }) => (
-  <TextFieldMui {...props} />
-))<Props>`
+const CustomTextField = styled(
+  ({ input, startAdornment, endAdornment, ...props }) => (
+    <TextFieldMui
+      {...props}
+      InputProps={{
+        startAdornment: startAdornment ? (
+          <InputAdornment position="start">{startAdornment}</InputAdornment>
+        ) : null,
+        endAdornment: endAdornment ? (
+          <InputAdornment position="start">{endAdornment}</InputAdornment>
```

Comment:
shouldn't this be `position="end"`?

Reply:
no really, have you watch it on storybook? that set the position inside the Adornment, not at the start or end of the input.

Reply:
actually, I tested it in the material-ui example. But behavior varies from what's in here.

![wrong-possition](https://user-images.githubusercontent.com/3315606/85905418-b925ac80-b7e1-11ea-86d0-2c15e127cf43.gif)